htmd/parameterization/cli.py

```diff
@@ -481,14 +481,14 @@ def _printEnergies(molecule, parameters, filename):
     energies = FFEvaluate(molecule, parameters).calculateEnergies(molecule.coords[:, :, 0])
 
     string = '''
-== Diagnostic Energies ==
+== Diagnostic Energies (in kcal/mol) ==
```

Comment:
Units at the end of the quantity, not in the header

Reply:
Any scientific paper disagrees with you, but as you wish

Reply:
Just keeping consistent with the other log messages